src/components/AppConfig/AppConfig.tsx

```diff
@@ -20,9 +20,35 @@ export interface LLMGatewaySettings {
   url?: string;
 }
 //////////////////////
+
```

Comment:
nit: you've added an api call function in the middle of a section that is declaring types/interfaces. There are a few other api calling functions at the bottom of the file, maybe move this down to that section of the file for a little more consistency.